pkg/util/winutil/shutil.go

```diff
@@ -74,6 +75,28 @@ func GetProgramDataDir() (path string, err error) {
 		// the path = windows.UTF16ToString... returns a
 		// go vet: "possible misuse of unsafe.Pointer"
 		path = windows.UTF16ToString((*[1 << 16]uint16)(unsafe.Pointer(retstr))[:])
+		path = filepath.Join(path, "Datadog")
+	}
+	return
+}
+
+// GetProgramDataDir returns the current programdatadir, usually
+// c:\programdata
```

Comment:
Nit but the example of this command should be `c:\programdata\Datadog\Datadog Agent` to be clearer.